tests/integration/test_recipe.py

```diff
@@ -1891,10 +1891,366 @@ def test_fx_vars_volcello_in_ofx_cmip6(tmp_path, patched_datafinder,
     fx_files = settings['fx_files']
     assert isinstance(fx_files, dict)
     assert len(fx_files) == 1
-    assert '_Ofx_' in fx_files['volcello']
+    assert '_Omon_' in fx_files['volcello']
+    assert '_Ofx_' not in fx_files['volcello']
```

Comment:
@schlunma note this change due to https://github.com/ESMValGroup/ESMValCore/pull/511#discussion_r383785460 - is this right?

Reply:
so you happy with the result @schlunma ? :beer:

Reply:
cheerios, dudios :beer: